### ISSUE TITLE
Alarm Clock crash when using sourcemod

### ISSUE THREAD

**Issue by hyperbolic2346** [jaredballou/insurgency-sourcemod#25]
Not sure what is going on here, but suddenly my insurgency server is crashing when someone joins with just the message "Alarm clock". Having trouble getting debug.log with -debug, but I do know that sourcemod is the issue through elimination. I synced up and got latest on everything and it is still crashing.

Have you seen this crash? Any idea what I might disable/remove to fix it or even how I can go about debugging it?

**Comment by jaredballou** [comment]
No idea. Have you made any headway, or is this still a problem?

**Comment by hyperbolic2346** [comment]
I haven't had much time to play with it, I just pulled the sourcemod stuff which I'm sorely lacking. All I can find are really old posts about how it was fixed in some patch.

**Comment by hyperbolic2346** [comment]
Possibly resurrecting this. I was playing with this tonight and right before the alarm clock I get what looks like an update. Any pointers to where the code is that is doing this or does this output give you any ideas?

```
Client "Player" connected (***.***.***.***:61032).
L 08/24/2016 - 02:30:11:   [2]  Successful: Yes
L 08/24/2016 - 02:30:11:
L 08/24/2016 - 02:30:11: SMC DEBUG
L 08/24/2016 - 02:30:11: information->baseurl
L 08/24/2016 - 02:30:11:   [0]  https://raw.github.com/jaredballou/insurgency-sourcemod/master
L 08/24/2016 - 02:30:11: version->latest
L 08/24/2016 - 02:30:11:   [0]  1.0.0
L 08/24/2016 - 02:30:11: information->notes
L 08/24/2016 - 02:30:11:   [0]  Event Logger
L 08/24/2016 - 02:30:11:   [1]  Log events to client or server
L 08/24/2016 - 02:30:11: files->source
L 08/24/2016 - 02:30:11:   [0]  Path_SM/scripting/events.sp
L 08/24/2016 - 02:30:11:   [1]  Path_SM/scripting/include/insurgency.inc
L 08/24/2016 - 02:30:11:   [2]  Path_SM/scripting/include/updater.inc
L 08/24/2016 - 02:30:11: files->plugin
L 08/24/2016 - 02:30:11:   [0]  Path_SM/plugins/events.smx
L 08/24/2016 - 02:30:11: END SMC DEBUG
L 08/24/2016 - 02:30:11:
L 08/24/2016 - 02:30:11: Download started:
L 08/24/2016 - 02:30:11:   [0]  URL: http://ins.jballou.com/sourcemod/update-damagemod.txt
L 08/24/2016 - 02:30:11:   [1]  Destination: addons/sourcemod/data/updater.txt
Alarm clock
```

It seems to get hung up on anything. First it was respawn, then botspawns. I kept moving them to the disabled directory thinking that maybe the script was the issue, but I think it's something else now.

**Comment by jaredballou** [comment]
I see you're also using my updated updater.smx plugin with BaseURL support. That's not quite ready, I probably should split those off to a develop branch rather than keep them in master. The output you posted is the debugging dump from my new updater. I took the upstream plugin, fixed all the issues with moving it to 1.8, and added a few new features. It should be harmless though.
